umake/ui/cli/__init__.py

```diff
@@ -157,6 +157,60 @@ def mangle_args_for_default_framework(args):
     return result_args
 
 
+def list_frameworks(listing, detailled=True):
```

Comment:
Multiple comments on this:
1. I will move that method in `umake/frameworks/__init__.py` (just above `load_frameworks()` definition) as this is a framework functionality
2. Do not forget to annotate the function explaining the parameters and functions
3. For i18n purpose and multi-possible UI in the future, this function would rather return some json dictionary like:
```
{ 'category_name':
  [
    { 'framework_name':
        {
           'detail': detail_name (always filled, then, the output will filter it if detail isn't chosen)
           'install_path': None or path string
           'installable': True or False
           'only_for_removal': True or False
        }
    },
 ]
}
```